### ISSUE TITLE
Install script contains adware

### ISSUE THREAD

**Issue by SephReed** [styled-components/styled-components#1645]
calling `npm install` has a postinstall adware script which outputs the following text:

```
Thanks for installing styled-components 🙏
                 Please consider donating to our open collective
                        to help us maintain this package.

                           Number of contributors: 186
                              Number of backers: 22
                             Annual budget: US$ 1,269
                             Current balance: US$ 687

     👉  Donate: https://opencollective.com/styled-components/donate
```

the adware script which injects the ad above should probably be removed.

It's understandable that you guys need money, but even if yours was the most important package in the folder this still would be unacceptable.

Can you imagine what it would be like if everyone did what you are doing here?  An entire console filled with advertisements?  

Please, recognize the golden rule, act as you wish others to act, and remove this adware.

**Comment by mxstbr** [comment]
Oh, I'm personally totally following the golden rule: I'd love to see everybody in open source make money! I'd love to see more ads in my terminal of developers trying to make a living doing what they love :100: (not an official opinion of the project)

**Comment by kitten** [comment]
Duplicate of https://github.com/styled-components/styled-components/issues/1590

Btw you can also call it the open-collective hook and not adware, as it has a name :wink:

Anyway, we discussed this before and would be totally happy to shorten this to a brief message with a link to our opencollective or removing it altogether.

cc @probablyup

**Comment by mxstbr** [comment]
In favor of making it shorter like nodemon!
